### ISSUE TITLE
OPA Dataset Construction

### ISSUE THREAD

**Issue by lingtianxia123** [bcmi/Object-Placement-Assessment-Dataset-OPA#5]
I am interested in understanding whether the OPA dataset was constructed exclusively using the MS COCO 'training' set or if the 'verification' set was also utilized in the process.  Was MS COCO 2014 used or MS COCO 2017 used?

**Comment by ustcnewly** [comment]
COCO 2014 train+val

**Comment by lingtianxia123** [comment]
Was the training set of COCO 2014 used to construct the OPA dataset, while the validation set of COCO 2014 was used to build the OPA test set?

**Comment by ustcnewly** [comment]
We use COCO 2014 train+val to construct the whole OPA dataset (OPA train, OPA test, OPA-ext). Then, we split the whole dataset into OPA train, OPA test, OPA-ext, in which OPA train and OPA test have no overlapped foreground/background. The split of OPA train/test has nothing to do with the split of COCO train/val.

**Comment by lingtianxia123** [comment]
Thank you for your response.
